doc/workflows/anaconda-webui.rst

```diff
@@ -0,0 +1,124 @@
+- **use_container** - Run npm command and the test itself inside podman container.
+- **port_ssh** - SSH port used for connection to the VM where Anaconda is running.
+- **port_webui** - Port used for connection to the Anaconda WebUI
+- **webui_location** - Part of URL where WebUI should be accessible.
+- **webui_startup_timeout** - Timeout in minutes for Anaconda WebUI to start after the VM gets IP
+- **webui_ssl_verify** - Verify SSL certificate used by Anaconda WebUI
+- **debug** - If true VMs is not removed at the end of the test
+
+VMHypervisors
+^^^^^^^^^^^^^
+Remote hypervisor URIs, example: qemu+ssh://hostname/system, one for each supported
+architecture.
+
+.. warning::
+    Remote hypervisors are not yet supported.
+
+
+AnacondaWebUIRepos
+^^^^^^^^^^^^^^^^^^
+Dictionary of git repositories with additional test scripts.
+
+AnacondaWebUIkernelCmdline
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Additional arguments for kernel cmdline, one for each supported architecture and
+one added to all architectures.
+
+Usage
+-----
+
+Testcase
+^^^^^^^^
+Execution type of this workflow is ``anaconda-webui``. Required automation_data
+are:
+
+- **script_file** - path to the test script, by default the base for this path
+  is anaconda git repository
+- **test_case** - python class in the script file (test script file can contain 
+  multiple test cases)
+
+Optional automation_data:
+
+- **test_repo** - Name of repository where the test script is located. Repositories
+  are defined in AnacondaWebUIRepos settings section. Sets the base path for script_file.
+- **additional_repos** - List of additional repos for installation (``inst.addrepo``), eg. ``['AppStream', 'CRB']``.
+- **kernel_cmdline** - Parameters to be added to kernel cmdline.
+- **webui_startup_timeout** - Override how long to wait for WebUI to start (in minutes).
+
+Example of minimal execution section for running test from anaconda repo::
+
+    execution:
+        type: anaconda-webui
+        automation_data:
+            script_file: ./ui/webui/test/integration/default.py
+            test_case: DefaultInstallation
+
+Example of execution section with all options set::
+
+    execution:
+        type: anaconda-webui
+        automation_data:
+            script_file: ./check-navigation.py
+            test_case: TestNavigation
+            test_repo: my-special-tests
+            additional_repos: ['AppStream', 'CRB']
+            kernel_cmdline: 'nosmt'
+            webui_startup_timeout: 15
+
+Hypervisor
+^^^^^^^^^^
+Currently only local hypervisor is supported.
+
+Test scripts
+^^^^^^^^^^^^
+Anaconda WebUI integration tests are python scripts that use cockpit's test framework
+and wrapper methods that make it easy to interact with the Web UI and run commands
+on the machine during installation and after reboot.
+
+Examples can be found in Anaconda Github repository:
+https://github.com/rhinstaller/anaconda/tree/master/ui/webui/test/integration
```

Comment:
As written above, it would be great to incorporate this into a simple but complete example.